src/cpuid/src/intel/normalize.rs

```diff
@@ -296,11 +296,211 @@ impl super::IntelCpuid {
 
     /// Update brand string entry
     fn update_brand_string_entry(&mut self) -> Result<(), NormalizeCpuidError> {
+        // Get host brand string.
+        let host_brand_string: [u8; BRAND_STRING_LENGTH] = crate::host_brand_string();
+
         let default_brand_string =
-            Self::default_brand_string().map_err(NormalizeCpuidError::GetBrandString)?;
+            default_brand_string(host_brand_string).map_err(NormalizeCpuidError::GetBrandString)?;
 
         self.apply_brand_string(&default_brand_string)
             .map_err(NormalizeCpuidError::ApplyBrandString)?;
         Ok(())
     }
 }
+
+/// Error type for [`IntelCpuid::default_brand_string`].
+#[derive(Debug, Eq, PartialEq, thiserror::Error)]
+pub enum DefaultBrandStringError {
+    /// Missing frequency.
+    #[error("Missing frequency: {0:?}.")]
+    MissingFrequency([u8; crate::BRAND_STRING_LENGTH]),
+    /// Missing space.
+    #[error("Missing space: {0:?}.")]
+    MissingSpace([u8; crate::BRAND_STRING_LENGTH]),
+    /// Insufficient space in brand string.
+    #[error("Insufficient space in brand string.")]
+    Overflow,
+}
+
+/// Gets the brand string always used for Intel.
```

Comment:
Should the description be more accurate as "Omit model specific information from CPU brand string?

Reply:
This will always display the Intel brand string as `Intel(R) Xeon(R) Processor @ ...` even when it is not a `Xeon` processor, it is not only omitting information here.

Reply:
Then: "Normalize brand string to a generic Xeon(R) processor, with the actual CPU frequency"

Reply:
Looks goods, fixed.